--- cmsis_gcc.h	2018-12-29 09:22:16.725343400 -0500
+++ cmsis_gcc.h	2018-12-29 07:08:40.984319100 -0500
@@ -147,7 +147,7 @@
  */
 __attribute__( ( always_inline ) ) __STATIC_INLINE uint32_t __get_PSP(void)
 {
-  register uint32_t result;
+  uint32_t result;
 
   __ASM volatile ("MRS %0, psp\n"  : "=r" (result) );
   return(result);
@@ -172,7 +172,7 @@
  */
 __attribute__( ( always_inline ) ) __STATIC_INLINE uint32_t __get_MSP(void)
 {
-  register uint32_t result;
+  uint32_t result;
 
   __ASM volatile ("MRS %0, msp\n" : "=r" (result) );
   return(result);
